lib/fizzbuzz.rb

```diff
@@ -1,13 +1,18 @@
 class SuperFizzBuzz
 
   def run(input)
-
-    #Implement your code here
-
+    if (input % 3 == 0 and input % 5 == 0)
+      "FizzBuzz"
+    elsif input % 5 == 0
+      "Buzz"
+    elsif input % 3 == 0
+      "Fizz"
+    else input
```

Comment:
We put the body of the `else` clause on its own line.

```suggestion
    else
      input
```

```ruby
if var.check?
  the_truthy_thing
else
  the_falsey_thing
end
```